pkg/controller/import-controller.go

```diff
@@ -332,10 +371,7 @@ func (r *ImportReconciler) createImporterPod(pvc *corev1.PersistentVolumeClaim)
 		return err
 	}
 	r.log.V(1).Info("Created POD", "pod.Name", pod.Name)
-	if requiresScratch {
```

Comment:
Why did this get removed? its important to actually create the scratch space. And I don't see an other place creating it.

Reply:
it is done anyway in updatePVCFromPod, so here in createPod it is done/tried first time, 
looks like I removed it during the thinking/experimenting process, but yes removal was not needed

Reply:
So this is here for the case where there is no pod yet, where we know ahead of time we need scratch space, like for upload and import from registry. so if we remove this code, then for upload and registry we have a create pod, determine it need scratch space there, delete pod, create scratch space, create pod. This just removes that cycle.

Reply:
In that case, I'll bring it back, but, when does the pod check for scratch - if it exists?
 
Because between creating the pod and the scratch - there is for sure a non zero time window, so I imagine on some clusters pod might start without having scratch ready. Can we create a scratch before pod (probably not because it is an owner)

Reply:
So the pod doesn't start unless all the PVCs it requires are available. So the pod will remain pending until the PVC is bound to a PV.